src/fetch-wrapper.ts

```diff
@@ -26,10 +25,10 @@ export default function fetchWrapper(
   let status: number;
   let url: string;
 
-  const fetch: typeof nodeFetch =
-    (requestOptions.request && requestOptions.request.fetch) ||
-    globalThis.fetch ||
-    /* istanbul ignore next */ nodeFetch;
+  let { fetch } = globalThis;
+  if (requestOptions.request?.fetch) {
+    fetch = requestOptions.request.fetch;
+  }
 
   return fetch(
```

Comment:
if neither `requestOptions.request.fetch` nor `globaltThis.fetch` is set, we should through a helpful error message.

Reply:
Added a new error in [93869aa](https://github.com/octokit/request.js/pull/580/commits/93869aae8bf019616a0385f691ccead11988e819)